st2actions/st2actions/runners/python_action_wrapper.py

```diff
@@ -62,6 +61,8 @@ def __init__(self, pack, file_path, parameters=None, parent_args=None):
             config.parse_args(args=self._parent_args)
         except Exception:
             pass
+        else:
+            db_setup()
```

Comment:
It's slightly weird that this is in the else block.

We probably also want to establish a connection if `config_parse_args()` fails since we ignore the exceptions right now and proceed normally even if it fails. Otherwise some weird issues will probably pop up due to connection not being established.